pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -21,10 +23,12 @@ import (
 const (
 	// DIDExchange did exchange protocol
 	DIDExchange        = "didexchange"
-	connectionSpec     = metadata.AriesCommunityDID + ";spec/connections/1.0/"
+	connectionSpec     = metadata.AriesCommunityDID + ";spec/didexchange/1.0/"
```

Comment:
is there a meaning to the prefix "connection" on these constants?

Reply:
Earlier didexchange protocol was named as connections. May be we can rename this to just spec